lib/util/InteractionOptionsWrapper.ts

```diff
@@ -161,7 +161,7 @@ export default class InteractionOptionsWrapper {
     }
 
     /**
-     * Get a channel option's complete channel. This will only succeed if the channel is cached. If the channel is a private channel and it isn't cached, a `InteractionResolvedChannel` instance will still be returned.
+     * Get a channel option's complete channel. This will only succeed if the channel is cached. If the channel is a private channel, and it isn't cached, a `InteractionResolvedChannel` instance will still be returned.
```

Comment:
```suggestion
     * Get a channel option's complete channel. This will only succeed if the channel is cached. If the channel is private and isn't cached, an `InteractionResolvedChannel` instance will still be returned.
```

The oxford comma is not required in writing, but this does have a few other issues that can be addressed